### PR TITLE
added some smartphones

### PR DESCRIPTION
**Pull request by ZeroOneDeveloper**
## added a few colors
- midnight
- starlight
- blue
- apple pink

## added some smartphones
- iPhone 13 mini
- iPhone 13
- iPhone 13 Pro
- iPhone 13 Pro Max
- Galaxy Fold 5G 3
- Galaxy Z Flip3 5G

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -118,4 +118,5 @@ dist
 .pnp.*
 
 build/
-.vscode
\ No newline at end of file
+.vscode
+.idea
```
Comment (wonderlandpark) [inline .gitignore]: Please do not change code not related to PR

File: src/lib/question.ts [hunk 2]
```diff
@@ -26,7 +26,7 @@ const Questions: Question[] = [
         value: true
       },
       {
-        label: '전 게임은 많이 플레이하지 않습니다.',
+        label: '전 게임은 많이 플레이하지 않습니다.',
```
Comment (wonderlandpark) [inline src/lib/question.ts]: This too

**Review by wonderlandpark — CHANGES_REQUESTED** [review]

**Review by wonderlandpark — CHANGES_REQUESTED** [review]
There is no color named `퍼시픽 플루` at iPhone 13 Pro (Max) model. I must be `시에라 블루`

**Comment by ZeroOneDeveloper** [comment]
I'm so sorry, I updated what you said, and I also updated the battery capacity that I wrote incorrectly.

**Review by wonderlandpark — APPROVED** [review]

**Pull request merged**